kuma/spam/akismet.py

```diff
@@ -25,6 +26,13 @@ def __str__(self):
                 (self.status_code, self.debug_help, self.response.text))
 
 
+def akismet_locale(language):
+    """
+    Convert a Django locale to a locale akismet expects. ("en-US" to "en_us")
+    """
+    return translation.to_locale(language).lower()
+
+
 class Akismet(object):
```

Comment:
This seems like it should be a method on a class. ``AkismetFormMixin`` or ``Akismet`` classes would work, but wouldn't be a natural fit since it isn't used there.  ``RevisionForm`` is the place it is actually used, so that feels like the best fit, but it does feel like generic code.

``RevisionForm`` is my preferred place, but I think I have a weak argument, so I'd support keeping it here for this round.